imagersite_project/imager_profile/models.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.dispatch import receiver
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+import uuid
+
+
+class ImagerProfile(models.Model):
+    # links this model to djangos user model and delete the instance of this model
+    # when the link django user moddle is deleted
+    profile_id = models.UUIDField(
+        primary_key=True,
+        default=uuid.uuid4,
+        editable=False)
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    camera_type = models.CharField('camera type', max_length=128, blank=True)
```

Comment:
This would be a good place to have a choice field instead of just a raw character field. There are only but so many camera types, and it would help for searchability if each type follows a prescribed format.